src/languages/en.js

```diff
@@ -265,8 +265,8 @@ export default {
         },
     },
     signInPage: {
-        expensifyDotCash: 'Expensify.cash',
-        expensifyIsOpenSource: 'The New Expensify is open source',
+        expensifyDotCash: 'New Expensify',
+        expensifyIsOpenSource: 'New Expensify is open source',
```

Comment:
Similar to my comment above, if we decide to use "the" at all, this seems like the other sensible place to do so.

Reply:
I don't think we want to use "the" here because this ends up as the header of the page.

Reply:
Oh, hold on, I was looking at the line above it for `expensifyDotCash`.

Reply:
The phrase for `expensifyIsOpenSource` is not referenced anywhere, so I will just remove it. I've updated this branch.